src/Components/Subscriptions/Table/index.jsx

```diff
@@ -71,78 +78,88 @@ const Table = ({ data }) => {
   };
 
   const closeModal = () => {
+    setShowConfirmDeleteModal(false);
     setShowDeleteModal(false);
+    history.goBack();
   };
 
   return (
-    <div>
-      <table className={styles.tableContainer}>
-        <thead>
-          <tr>
-            <th>Classes</th>
-            <th>Members</th>
-            <th>Date</th>
-          </tr>
-        </thead>
-        <tbody>
-          {tableData.length > 0 ? (
-            tableData.map((subscription) => (
-              <tr className={styles.line} key={subscription._id}>
-                <td
-                  className={`${styles.item} ${styles.td}`}
-                >{`${subscription.classes.day} ${subscription.classes.time}`}</td>
-                {!subscription.members ? (
-                  <td className={`${styles.item} ${styles.td}`}>{'empty'}</td>
-                ) : (
+    <Router>
+      <div className={styles.container}>
+        <table className={styles.tableSubscription}>
+          <thead className={styles.containerThead}>
+            <tr className={styles.thead}>
+              <th>Classes</th>
+              <th>Members</th>
+              <th>Date</th>
+              <th colSpan="2"></th>
+            </tr>
+          </thead>
+          <tbody>
+            {tableData.length > 0 ? (
+              tableData.map((subscription) => (
+                <tr key={subscription._id} className={styles.item}>
+                  <td>{`${subscription.classes.day} ${subscription.classes.time}`}</td>
+                  {!subscription.members ? (
+                    <td>{'empty'}</td>
+                  ) : (
+                    <td>{`${subscription.members.name} ${subscription.members.lastName}`}</td>
+                  )}
+                  <td>{formatDate(subscription.date)}</td>
                   <td
-                    className={`${styles.item} ${styles.td}`}
-                  >{`${subscription.members.name} ${subscription.members.lastName}`}</td>
-                )}
-                <td className={`${styles.item} ${styles.td}`}>{formatDate(subscription.date)}</td>
-                <td
-                  className={`${styles.btnEdit} ${styles.td}`}
-                  onClick={() => handleEdit(subscription._id)}
-                >
-                  <img src={imgEditSubscription}></img>
-                </td>
-                <td
-                  className={`${styles.btnDelete} ${styles.td}`}
-                  onClick={() => handleConfirmDelete(subscription._id)}
-                >
-                  <img src={imgDeleteSubscription}></img>
-                </td>
+                    onClick={() => handleEdit(subscription._id)}
+                    className={`${styles.itemButton} ${styles.itemButtonEdit}`}
+                  >
+                    <Link to={`/subscription/edit/${subscription._id}`}>
```

Comment:
i did it like this: subscription/form for both (create and update) but maybe is something to discuss with the entire group